x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/vector/VectorSimilarityFunctionsIT.java

```diff
@@ -192,8 +192,6 @@ private static float[] readVector(List<Float> leftVector) {
 
     @Before
     public void setup() throws IOException {
-        assumeTrue("Dense vector type is disabled", EsqlCapabilities.Cap.DENSE_VECTOR_FIELD_TYPE.isEnabled());
```

Comment:
do we want to check for the other EsqlCapabilities for the dense vector similarity functions?
asking because these should run in non snapshot builds too, where the vector similarity functions are not yet available.
Also it might not hurt to run the release tests, if not in CI, at least locally.